modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/paths/Testray.path

```diff
@@ -40,6 +40,11 @@
 	<td>//div[contains(@class, 'dropdown-menu show')]//button[contains(text(), "${key_value} Items")]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>LINK_WITH_TEXT</td>
+	<td>//a[contains(@href,'#/') and contains(text(),'${key_text}')]</td>
+	<td></td>
+</tr>
```

Comment:
What do you think about changing the title of the path to: LINK_TO_HOME because you are referencing where the path.

Reply:
Done